addons/web/static/src/search/cog_menu/cog_menu.js

```diff
@@ -30,6 +30,7 @@ export class CogMenu extends ActionMenus {
         context: { type: ActionMenus.props.context, optional: true },
         resModel: { type: ActionMenus.props.resModel, optional: true },
         items: { ...ActionMenus.props.items, optional: true },
+        slots: { type: Object, optional: true },
```

Comment:
That's weird that the prop is defined in the parent... but not used in its template... but used in the child's template :thinking: 
I would have done one or the other, but not the "in-between" the two.

Reply:
Each xpath was identical, so I moved it to the parent component directly. Right before the `t-if="state.printItems.length"`